### PR TITLE
Progress #822 - Adding DashSpeed to ObjAiBase

### PR DESCRIPTION
**Pull request by abosma**
Improved the state of dashing by implementing DashSpeed to ObjAiBase and returning it if the character is currently dashing.

Progress #822

### CODE REVIEW

**Comment by cntseesharp** [comment]
Champion can have several dashes (Lee Sin, J4, Kha E range difference, Riven), thus dash speed will be different

File: GameServerLib/GameObjects/AttackableUnits/AI/ObjAIBase.cs [hunk 1]
```diff
@@ -51,6 +51,7 @@ public class ObjAiBase : AttackableUnit, IObjAiBase
         public bool IsCastingSpell { get; set; }
         public bool IsMelee { get; set; }
         public bool IsDashing { get; protected set; }
+        public float DashSpeed { get; set; }
```
Comment (danil179) [inline GameServerLib/GameObjects/AttackableUnits/AI/ObjAIBase.cs:54]: I think we can start with at least protected set and see if "public" is really needed.

**Review by danil179 — CHANGES_REQUESTED** [review]
One comment.

**Review by danil179 — APPROVED** [review]
Will open issue about protected vs public.

**Pull request merged**